### ISSUE TITLE
incompatible on SAMSUNG MZVLB1T0HBLR-00000?

### ISSUE THREAD

**Issue by juliusagodzo** [Chilluminati91/Huawei-Matebook-X-Pro-2020-Hackintosh#7]
hi, this EFI is not working on the 1TB ssd model.
the ssd model is SAMSUNG MZVLB1T0HBLR-00000

do you know that? 
are there any fix? because I tried to install but the installation failed and reboot the pc damaging the ssd partition table

**Comment by Rino53** [comment]
Also waiting for this fix

**Comment by juliusagodzo** [comment]
> Also waiting for this fix

are you facing the same problem?

**Comment by Rino53** [comment]
> > Also waiting for this fix
> 
> are you facing the same problem?

No, but i have same model with 1tb samsung drive. Cant use my mbpx as testing stand right now.

**Comment by happyDemon** [comment]
https://github.com/acidanthera/NVMeFix
could be of help

someone was successful when installing macOS 10.15.3
https://github.com/acidanthera/bugtracker/issues/843

**Comment by juliusagodzo** [comment]
> https://github.com/acidanthera/NVMeFix
> could be of help
> 
> someone was successful when installing macOS 10.15.3
> [acidanthera/bugtracker#843](https://github.com/acidanthera/bugtracker/issues/843)

tried but not working.
the owner of this repo is not replying. i think that those issue need to be solved by cooperating whit each other otherwise this hackintosh will die soon.

**Comment by juliusagodzo** [comment]
> Also waiting for this fix

hi, have you find any solution?

**Comment by mattmatt0** [comment]
Hi,
It seems to me that the Samsung SSD is indeed the issue. I couldn't install macOS on the internal drive - the installer would keep crashing. However, installing it on a separate drive (plugged via one of the USB-C ports) worked just fine.

**Comment by Jensilein** [comment]
Hi there, I can confirm that issue. Installation of macOS wasn‘t possible. After having swapped the stock Samsung SSD MZVLB1T0HBLR-00000 by a WD Black SN770 1TB SSD everything worked like a charm.
